app/models/comment.rb

```diff
@@ -0,0 +1,9 @@
+class Comment < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+  afetr_save :update_comment_counter
+
+  def update_comment_counter
+    post.update(CommentsCounter: count)
+  end
```

Comment:
- [ ] The `count` variable you are using here is undefined, Please make sure you use valid code to update the counter. I would suggest you to using the below code.

```ruby
def update_comment_counter
    post.increment!(:CommentsCounter)
  end
```

- [ ] [Optional] I would suggest you make this method private So it can't be accessed outside of this class.

Reply:
thank you . just sharing my thoughts was using self.count so linter trimmed self off